README.md

```diff
@@ -41,24 +41,24 @@ jobs:
     steps:
       - name: Check out repository
         uses: actions/checkout@v2
-      - name: Install MATLAB
-        uses: matlab-actions/setup-matlab@v0
+      - name: Set up MATLAB
+        uses: matlab-actions/setup-matlab@v1
       - name: Run tests and generate artifacts
-        uses: matlab-actions/run-tests@v0
+        uses: matlab-actions/run-tests@v1
         with:
           test-results-junit: test-results/results.xml
           code-coverage-cobertura: code-coverage/coverage.xml
 ```
 
 ## Set Up MATLAB
-When you define your workflow in the `.github/workflows` directory of your repository, specify the **Set Up MATLAB** action as `matlab-actions/setup-matlab@v0`. The action accepts an optional input.
+When you define your workflow in the `.github/workflows` directory of your repository, specify the **Setup MATLAB** action as `matlab-actions/setup-matlab@v1`. The action accepts an optional input.
 
 | Input     | Description |
 |-----------|-------------|
-| `release` | (Optional) MATLAB release to install. You can specify R2020a or a later release. If you do not specify `release`, the action installs the latest release of MATLAB.<br/>**Example**: `R2020a`
+| `release` | (Optional) MATLAB release to set up. You can specify R2020a or a later release. If you do not specify `release`, the action sets up the latest release of MATLAB.<br/>**Example**: `R2020a`
```

Comment:
```suggestion
| `release` | (Optional) MATLAB release to set up. You can specify R2020a or a later release. If you do not specify `release`, the action sets up the latest release of MATLAB.<br/>**Example**: `R2021a`
```

I like always like promoting our latest release when we can :)

Reply:
+1

Reply:
Me too; I directly committed your suggestion (neato feature in GitHub I must say 😄)